cmd/observe/observe.go

```diff
@@ -211,6 +211,10 @@ more.`,
 		"to-identity", ofilter,
 		"Show all flows terminating at an endpoint with the given security identity"))
 
+	observerCmd.Flags().Var(filterVar(
+		"node-names", ofilter,
+		`Show all flows which match the given node names (e.g. "k8s*", "test-cluster/*.company.com")`)
```

Comment:
i think this line is missing a closing `)` which is what is making the linter freak out

Reply:
Good catch, thanks!